src/middleware/error-handler.js

```diff
@@ -1,16 +1,21 @@
-const logger = require('../utils/logger');
+const { logger } = require("../utils/logger");
 
-const errorHandler = (
-  err,
-  req,
-  res,
-  next
-) => {
-  logger.error(err.message);
-  if (!err.statusCode) err.statusCode = 500;
-  res.status(err.statusCode).send(err.message);
+const ERROR_CODES = {
+  UNPROCESSABLE_ENTITY: 422,
+  NOT_FOUND: 404,
+  INTERNAL_SERVER_ERROR: 500
+};
+
+const errorHandler = (err, req, res, next) => {
+  console.log("raw", err);
+  const error = JSON.parse(err.message);
+  console.log("error", error);
+  logger.error(error.status);
+  if (!error.status) error.status = ERROR_CODES.INTERNAL_SERVER_ERROR;
```

Comment:
```suggestion
  error.status = error.status || ERROR_CODES.INTERNAL_SERVER_ERROR;
```
Is a more idiomatic way of doing this.  In general, you should try to avoid one-line if-statements like this. They look very different with properly blocked-out if-statements, so it's just a bit harder to read. It's also just a little harder to expand later on (because you need to fix indentation etc first).  So in general, if you need an if, put the body on it's own line with { } surrounding it.

Reply:
Thanks!